examples/lang/erc721/src/lib.rs

```diff
@@ -76,11 +76,14 @@ contract! {
     struct Erc721 {
         token_owner: storage::HashMap<TokenId, AccountId>,
         token_approvals: storage::HashMap<TokenId, AccountId>,
-        owned_tokens_count: storage::HashMap<AccountId, Counter>
+        owned_tokens_count: storage::HashMap<AccountId, Counter>,
+        symbol: storage::Value<u8>,
```

Comment:
Why do we need symbol? I though we discussed this before and concluded that we won't need it for a basic Erc721 implementation, didn't we? Also as far as I see it this symbol serves as UI element so we won't want to push for that.

Reply:
Removed. This was a fix for the ABI error.

Reply:
It turns out that it needs to initialize a variable on deploy or it breaks the ABI generation.